src/System.Linq.Expressions/src/System/Linq/Expressions/ExpressionStringBuilder.cs

```diff
@@ -632,43 +595,29 @@ protected internal override Expression VisitUnary(UnaryExpression node)
         {
             switch (node.NodeType)
             {
-                case ExpressionType.TypeAs:
```

Comment:
For the reviewer's convenience, this table is now ordered in the same was as `MakeUnary` over here: https://github.com/dotnet/corefx/blob/8960aac11e908e59573fafbdeff66789db4c338d/src/System.Linq.Expressions/src/System/Linq/Expressions/UnaryExpression.cs#L337